docs/documentos-projeto/identidade.md

```diff
@@ -0,0 +1,95 @@
+
+![Header](../assets/identidadeVisual/Header.png)
+
+&emsp;&emsp;Existem variações de cabeçalho para adaptação. Os outros estilos podem ser visto na [documentação](https://www.gov.br/ds/components/header?tab=desenvolvedor)
+
+### 3.2 Rodapé
+
+&emsp;&emsp;Geralmente localizado na parte inferior das páginas, o footer (rodapé) pode ser organizado de formas distintas. Normalmente essa organização é definida através da combinação de elementos que reforçam a identidade visual com o conteúdo a ser informado, respeitando os objetivos de negócio e as necessidades do usuário.
+
+![Footer](../assets//identidadeVisual/Footer.png)
+
+&emsp;&emsp;Existe uma variação quanto ao tema que também pode ser utilizada, que no caso o background color é branco. Outras variações, detalhes de como manipular as classes e o estilo do componente oferecido pelo próprio governo podem ser vistos na [documentação](https://www.gov.br/ds/components/footer?tab=desenvolvedor)
+
+### 3.3 Paleta de Cores
+
+&emsp;&emsp;A aplicação correta da paleta de cores visa trazer consistência ao Design System e contribuir para garantir uma interface unificada e facilmente reconhecível pelos seus usuários. É fundamental que a paleta esteja alinhada aos Princípios do Design System e seja aplicada corretamente na biblioteca de componentes contribuindo para o equilíbrio entre os diversos produtos e sistemas do governo.
+
+![Cores](../assets/identidadeVisual/Cores.png)
+
+&emsp;&emsp;Pela [documentação](https://www.gov.br/ds/fundamentos-visuais/cores) é possível ver detalhadamente quais cores devem ser utilizadas e quais as suas funções específicas, portanto é indicado que seja utilizado como guia essas cores para seguir o padrão mínimo esperado
+
+### 3.4 Tipografia
+
+&emsp;&emsp;A tipografia utilizada no Design System é a 'Rawline' utilizando suas diferentes aplicações de peso, o que garante uma hierarquia entre os diferentes textos. Para saber mais detalhes sobre a utilização da tipografia e suas variações, acesse a [documentação](https://www.gov.br/ds/fundamentos-visuais/tipografia)
+
+### 3.5 Iconografia
+
+&emsp;&emsp;A coleção de ícones escolhida pelo Design System do governo é a 'Font Awesome' por possuir uma boa variedade de ícones com layout simples, sintético e amigável
+
+### 3.6 Botões
+
+&emsp;&emsp;Os botões são elementos interativos da interface, que permitem que os usuários acessem funcionalidades, executem ações ou naveguem pela interface.
+
+![Botão1](../assets/identidadeVisual/Botão1.png)
+![Botão2](../assets//identidadeVisual/Botão2.png)
+![Botão3](../assets/identidadeVisual/Botão3.png)
+
+&emsp;&emsp;Propriedades adicionais, estados do botão e outros detalhes podem ser vistos na [documentação](https://www.gov.br/ds/components/button?tab=desenvolvedor)
+
+## Referências
+
+> EQUIPE DNIT 2023-1. Identidade Visual. Disponível em: <https://fga-eps-mds.github.io/2023-1-Dnit-DOC/identidadeVisual/identidadeVisual/#assinatura-conjunta>
```

Comment:
Correção seguindo a ABNT
```suggestion
> EQUIPE DNIT 2023-1. Identidade Visual. GitHub, 2023. Disponível em: <https://fga-eps-mds.github.io/2023-1-Dnit-DOC/identidadeVisual/identidadeVisual/#assinatura-conjunta>. Acesso em 04 out 2023.
```

Reply:
Recomendo fazer essa correção de referência seguindo a ABNT para os demais documentos: 
SOBRENOME, Nome(*) . Título da matéria ou página. Nome do site, ano. Disponível em: &lt;URL&gt;. Acesso em: dia mês (abreviado) ano.

(*) Se o autor não for identificado, comece a referência pelo título.

Reply:
Sobre o padrão, acredito que, por hora, é melhor continuarmos com esse padrão mais simples e que atende bem.